core-web/libs/template-builder/src/lib/components/template-builder/components/add-style-classes-dialog/add-style-classes-dialog.component.scss

```diff
@@ -0,0 +1,53 @@
+@use "variables" as *;
+:host {
+    width: 33.875rem;
+    display: block;
+}
+
+.dialog__auto-complete-container {
+    padding: 0.375rem $spacing-7 $spacing-1;
+    overflow: auto;
+    max-height: 5.375rem;
+
+    .auto-complete-container__title {
+        margin: 0;
+        padding-bottom: 0.375rem;
+        color: $black;
+    }
+}
+
+.dialog__actions-container {
+    display: flex;
+    justify-content: end;
+    padding: 1.25rem $spacing-5;
+}
+
+::ng-deep {
+    .p-autocomplete-multiple-container {
+        gap: $spacing-1;
+        margin: 0;
+
+        .p-autocomplete.p-autocomplete-multiple
+            .p-autocomplete-multiple-container
+            li.p-autocomplete-token {
+            margin-right: 0;
+        }
+    }
+
+    .p-dialog .p-dialog-content {
+        padding: 0;
+    }
+
+    .p-dialog {
+        max-height: 16.375rem;
+
+        .p-dialog-header {
+            padding: 1.75rem $spacing-5;
+
+            .p-dialog-title {
+                font-size: $font-size-xl;
+                line-height: 31px;
+            }
+        }
+    }
+}
```

Comment:
Not sure why we need all of this... or if we have to move to global.

Reply:
This is specific for this design, I don't think this should be on the global styles